lib/common.bash

```diff
@@ -281,3 +281,36 @@ restart_docker_service() {
 
 	return 0
 }
+
+# This function verifies that the dockerfile version is
+# equal to the test version in order to build the image or
+# just run the test
+check_dockerfiles_images()
+{
+	local image="$1"
```

Comment:
Might be best to set this to default to handle the case where this script is sourced and the calling script has `set -o nounset`:

```suggestion
	local image="${1:-}"
```

Same comment for the variable below.